packages/contentful/migrations/news/20230213120700-add-publish-date-field.js

```diff
@@ -0,0 +1,24 @@
+module.exports.description = 'Adds publishDate field';
+
+module.exports.up = function (migration) {
+  const news = migration.editContentType('news');
+  news
+    .createField('publishDate')
+    .name('Publish Date')
+    .type('Date')
+    .localized(false)
+    .required(true)
+    .validations([])
+    .disabled(false)
+    .omitted(false);
```

Comment:
can we add a default to today?

Reply:
hmm good idea, let me test that

Reply:
Trying to add default value here https://github.com/yldio/asap-hub/pull/2603/commits/733a373e0ad67aadbb701e2d8d2574206ab05149

It becomes a snapshot of the date when the migration was created 

![schema](https://user-images.githubusercontent.com/16595804/219028390-d0b31824-1ee2-4d71-89ec-3360c6c95c99.png)

![content-create](https://user-images.githubusercontent.com/16595804/219028405-0f83aee9-babd-4762-884a-bd193a909ca0.png)

I will revert this last commit